RELEASE-NOTES.txt

```diff
@@ -6,6 +6,7 @@
 - [*] Fix: top performers in "This Week" tab should be showing the same data as in WC Admin.
 - [*] Fix: visitor stats in Dashboard should be more consistent with web data on days when the end date for more than one tab is the same (e.g. "This Week" and "This Month" both end on January 31). [https://github.com/woocommerce/woocommerce-ios/pull/3532]
 - [internal] Refactored Core Data migrator stack to help reduce crashes [https://github.com/woocommerce/woocommerce-ios/pull/3523]
+- [**] Due to popular demand, the product SKU is displayed once again in Order Details screen. [https://github.com/woocommerce/woocommerce-ios/pull/3564]
```

Comment:
super nit: we generally put the higher priority release notes at the top so that the list is sorted by priority (`[***]` --> `[**]` --> `[*]` --> `[internal]`) 😄

Reply:
Ahh! That's good to know! I fixed the sorting in 94cf380. Thanks for the review @jaclync 😄